From 250ceac258ec47aeb549c5620f265bd67bd35595 Mon Sep 17 00:00:00 2001
From: Ruixuan Gu <quic_ruixgu@quicinc.com>
Date: Thu, 15 Jun 2023 22:34:13 +0800
Subject: [PATCH] Makefile patch for rtsp-alg

Signed-off-by: Ruixuan Gu <quic_ruixgu@quicinc.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 1bf7650..338271f 100644
--- a/Makefile
+++ b/Makefile
@@ -11,13 +11,13 @@ else
 
 KERNELDIR := /lib/modules/`uname -r`/build
 all: 
-	$(MAKE) -C $(KERNELDIR) M=`pwd` modules
+	$(MAKE) -C $(KERNEL_SRC) M=$(shell pwd) modules
 
 debug:
-	$(MAKE) -C $(KERNELDIR) EXTRA_CFLAGS=-DDEBUG M=`pwd` modules
+	$(MAKE) -C $(KERNEL_SRC) EXTRA_CFLAGS=-DDEBUG M=$(shell pwd) modules
 
 modules_install:
-	$(MAKE) -C $(KERNELDIR) M=`pwd` modules_install
+	$(MAKE) -C $(KERNEL_SRC) M=$(shell pwd) modules_install
 
 clean:
 	rm -rf *.o *.ko *.mod.c .*.cmd Module.symvers modules.order .tmp_versions
-- 
2.37.3.windows.1

