--- a/ddclient	2019-10-16 20:07:54.636050900 +0530
+++ b/ddclient	2019-10-25 18:23:34.139728937 +0530
@@ -1027,7 +1027,7 @@
     # Check for only owner has any access to config file
     my ($dev, $ino, $mode, @statrest) = stat(FD);
     if ($mode & 077) {
-	if (-f FD && (chmod 0600, $file)) {
+	if (-f FD && (chmod 0664, $file)) {
            warning("file $file must be accessible only by its owner (fixed).");
        } else {
            # fatal("file $file must be accessible only by its owner.");
