From 5aac60c6f6fdb11083055ab02e2b81950596fcd5 Mon Sep 17 00:00:00 2001
From: Shimeng Huang <quic_shimhuan@quicinc.com>
Date: Wed, 22 Feb 2023 10:57:39 +0800
Subject: [PATCH] dhcpcdv6_reset_RA

---
 src/if-linux.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/if-linux.c b/src/if-linux.c
index fd05147b..51353e57 100644
--- a/src/if-linux.c
+++ b/src/if-linux.c
@@ -2118,9 +2118,6 @@ if_setup_inet6(const struct interface *ifp)
 		 * error as such so just log it and continue */
 		if (errno != ENOENT)
 			logerr("%s: %s", __func__, path);
-	} else if (ra != 0) {
-		if (if_writepathuint(ctx, path, 0) == -1)
-			logerr("%s: %s", __func__, path);
 	}
 }
 
-- 
2.17.1

