From dd0090eac13fc7d119d44cf191fd4eb9c777aef0 Mon Sep 17 00:00:00 2001
From: James Wyatt Guidry <quic_jguidry@quicinc.com>
Date: Mon, 6 Jun 2022 10:24:04 -0700
Subject: [PATCH 1/1] pugixml: Modify source Makefile for OpenWRT compilation

Signed-off-by: James Wyatt Guidry <quic_jguidry@quicinc.com>
---
 Makefile | 107 ++++++++++-----------------------------------------------------
 1 file changed, 17 insertions(+), 90 deletions(-)

diff --git pugixml-1.11.4.orig/Makefile pugixml-1.11.4/Makefile
index 028641b..b5cfbe1 100644
--- pugixml-1.11.4.orig/Makefile
+++ pugixml-1.11.4/Makefile
@@ -1,99 +1,26 @@
-.SUFFIXES:
-MAKEFLAGS+=-r
+# Makefile for OpenWRT system
 
-config=debug
-defines=standard
-cxxstd=c++11
-# set cxxstd=any to disable use of -std=...
+CFLAGS += -Wundef \
+	  -Wno-trigraphs \
+	  -g -O0 \
+	  -fno-inline \
+	  -fno-short-enums \
+	  -fpic
 
-BUILD=build/make-$(CXX)-$(config)-$(defines)-$(cxxstd)
+RM := rm -rf
 
-SOURCES=src/pugixml.cpp $(filter-out tests/fuzz_%,$(wildcard tests/*.cpp))
-EXECUTABLE=$(BUILD)/test
+LIBPUGIXML := libpugixml.so
 
-VERSION=$(shell sed -n 's/.*version \(.*\).*/\1/p' src/pugiconfig.hpp)
-RELEASE=$(filter-out scripts/archive.py docs/%.adoc,$(shell git ls-files docs scripts src CMakeLists.txt LICENSE.md readme.txt))
+LIBPUGIXML_SRCS = src/pugixml.cpp
 
-CXXFLAGS=-g -Wall -Wextra -Werror -pedantic -Wundef -Wshadow -Wcast-align -Wcast-qual -Wold-style-cast -Wdouble-promotion
-LDFLAGS=
+LIBPUGIXML_OBJS := $(LIBPUGIXML_SRCS:.cpp=.o)
 
-ifeq ($(config),release)
-	CXXFLAGS+=-O3 -DNDEBUG
-endif
-
-ifeq ($(config),coverage)
-	CXXFLAGS+=-coverage
-	LDFLAGS+=-coverage
-endif
-
-ifeq ($(config),sanitize)
-	CXXFLAGS+=-fsanitize=address,undefined -fno-sanitize=float-divide-by-zero,float-cast-overflow -fno-sanitize-recover=all
-	LDFLAGS+=-fsanitize=address,undefined
-endif
-
-ifeq ($(config),analyze)
-	CXXFLAGS+=--analyze
-endif
-
-ifneq ($(defines),standard)
-	COMMA=,
-	CXXFLAGS+=-D $(subst $(COMMA), -D ,$(defines))
-endif
-
-ifneq ($(findstring PUGIXML_NO_EXCEPTIONS,$(defines)),)
-	CXXFLAGS+=-fno-exceptions
-endif
-
-ifneq ($(cxxstd),any)
-	CXXFLAGS+=-std=$(cxxstd)
-endif
-
-OBJECTS=$(SOURCES:%=$(BUILD)/%.o)
-
-all: $(EXECUTABLE)
-
-ifeq ($(config),coverage)
-test: $(EXECUTABLE)
-	-@find $(BUILD) -name '*.gcda' -exec rm {} +
-	./$(EXECUTABLE)
-	@gcov -b -o $(BUILD)/src/ pugixml.cpp.gcda | sed -e '/./{H;$!d;}' -e 'x;/pugixml.cpp/!d;'
-	@find . -name '*.gcov' -and -not -name 'pugixml.cpp.gcov' -exec rm {} +
-	@sed -i -e "s/#####\(.*\)\(\/\/ unreachable.*\)/    1\1\2/" pugixml.cpp.gcov
-else
-test: $(EXECUTABLE)
-	./$(EXECUTABLE)
-endif
-
-fuzz_%: $(BUILD)/fuzz_%
-	@mkdir -p build/$@
-	$< build/$@ tests/data_fuzz_$* -max_len=1024 -dict=tests/fuzz_$*.dict
+$(LIBPUGIXML): $(LIBPUGIXML_OBJS)
+	$(CC) $(LIBPUGIXML_OBJS) $(CFLAGS) -shared -o $@
+	mkdir -p src/lib
+	mv libpugixml.so src/lib/
 
 clean:
-	rm -rf $(BUILD)
-
-release: build/pugixml-$(VERSION).tar.gz build/pugixml-$(VERSION).zip
-
-docs: docs/quickstart.html docs/manual.html
-
-build/pugixml-%: .FORCE | $(RELEASE)
-	@mkdir -p $(BUILD)
-	TIMESTAMP=`git show v$(VERSION) -s --format=%ct` && python3 scripts/archive.py $@ pugixml-$(VERSION) $$TIMESTAMP $|
-
-$(EXECUTABLE): $(OBJECTS)
-	$(CXX) $(OBJECTS) $(LDFLAGS) -o $@
-
-$(BUILD)/fuzz_%: tests/fuzz_%.cpp src/pugixml.cpp
-	@mkdir -p $(BUILD)
-	$(CXX) $(CXXFLAGS) -fsanitize=address,fuzzer $^ -o $@
-
-$(BUILD)/%.o: %
-	@mkdir -p $(dir $@)
-	$(CXX) $< $(CXXFLAGS) -c -MMD -MP -o $@
-
--include $(OBJECTS:.o=.d)
-
-.SECONDEXPANSION:
-docs/%.html: docs/%.adoc $$(shell sed -n 's/include\:\:\(.*\)\[.*/docs\/\1/p' docs/%.adoc)
-	asciidoctor -b html5 -a version=$(VERSION) $< -o $@
+	$(RM) lib/
+	$(RM) *.o
 
-.PHONY: all test clean release .FORCE
-- 
2.7.4

