From 186385d6043c5e119f4fcecf543d3ab4850156c8 Mon Sep 17 00:00:00 2001
From: Gustavo Solaira <gustavos@codeaurora.org>
Date: Tue, 6 Aug 2019 10:31:41 -0700
Subject: [PATCH] Fix compilation in Yocto

---
 tc/tc_core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tc/tc_core.c b/tc/tc_core.c
index 46eaefb..5a749d3 100644
--- a/tc/tc_core.c
+++ b/tc/tc_core.c
@@ -10,6 +10,7 @@
  *
  */
 
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
