From 86686868032d293c92f0d96c25ea376eec76f54a Mon Sep 17 00:00:00 2001
From: Mayank Vishwari <quic_mayankvi@quicinc.com>
Date: Fri, 20 Dec 2024 10:16:38 +0530
Subject: [PATCH] Enable conntrack for dnsmasq

---
 src/config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.h b/src/config.h
index e722e98..2a0e8ca 100644
--- a/src/config.h
+++ b/src/config.h
@@ -199,7 +199,7 @@ RESOLVFILE
 /* #define HAVE_DBUS */
 /* #define HAVE_IDN */
 /* #define HAVE_LIBIDN2 */
-/* #define HAVE_CONNTRACK */
+#define HAVE_CONNTRACK
 /* #define HAVE_CRYPTOHASH */
 /* #define HAVE_DNSSEC */
 /* #define HAVE_NFTSET */
-- 
2.25.1

