From 526c550ac32bf461718932b8708d8744d25f2ecd Mon Sep 17 00:00:00 2001
From: Ganesh Babu Kamma <quic_gkamma@quicinc.com>
Date: Wed, 21 Feb 2024 14:40:07 +0530
Subject: [PATCH] Enable conntrack for dnsmasq

---
 src/config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.h b/src/config.h
index e722e98..2a0e8ca 100644
--- a/src/config.h
+++ b/src/config.h
@@ -199,7 +199,7 @@ RESOLVFILE
 /* #define HAVE_DBUS */
 /* #define HAVE_IDN */
 /* #define HAVE_LIBIDN2 */
-/* #define HAVE_CONNTRACK */
+#define HAVE_CONNTRACK
 /* #define HAVE_CRYPTOHASH */
 /* #define HAVE_DNSSEC */
 /* #define HAVE_NFTSET */
