From 1007079c6d4d4c116809cf6caad78a804b596695 Mon Sep 17 00:00:00 2001
From: Dheeraj Kumar <quic_dhekum@quicinc.com>
Date: Mon, 1 May 2023 12:45:34 +0530
Subject: [PATCH] Enable conntrack for dnsmasq

---
 src/config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.h b/src/config.h
index 7d08f7d..f78f43b 100644
--- a/src/config.h
+++ b/src/config.h
@@ -184,7 +184,7 @@ RESOLVFILE
 /* #define HAVE_DBUS */
 /* #define HAVE_IDN */
 /* #define HAVE_LIBIDN2 */
-/* #define HAVE_CONNTRACK */
+#define HAVE_CONNTRACK
 /* #define HAVE_DNSSEC */
 
 
-- 
2.7.4

