From 92ffa38ed9b7314ff597f7103a8f9b7fcdc79694 Mon Sep 17 00:00:00 2001
From: Mayank Vishwari <quic_mayankvi@quicinc.com>
Date: Fri, 20 Dec 2024 10:39:11 +0530
Subject: [PATCH] Include libnetfilter_conntrack lib for dnsmasq

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a9bcab9..01d73f9 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ CFLAGS        = -Wall -W -O2
 LDFLAGS       = 
 COPTS         = 
 RPM_OPT_FLAGS = 
-LIBS          = 
+LIBS          = -lnetfilter_conntrack
 LUA           = lua
 
 #################################################################
-- 
2.25.1

